create-sbuild-chroot-schroot.py

```diff
@@ -90,6 +90,10 @@ def get_sbuild_architecture():
 parser.add_argument("distribution", type=str, nargs="*")
 args = parser.parse_args()
 
+if not os.path.exists(args.config):
+    parser.error(f"config path {args.config} does not exist")
+
+logging.info("Reading configuration")
```

Comment:
Can we drop the info message? It doesn't really provide much value, and clutters the output.

Reply:
Drop or convert to debug messages?

Reply:
Done